src/Components/Homepage.jsx

```diff
@@ -0,0 +1,28 @@
+import '../index.css';
+import { NavLink } from 'react-router-dom';
+
+const Homepage = () => (
+  <div className="bgcontainer">
+    <div className="getposition">
+      <div className="flex justify-center  items-center">
+        <h1 className="text-3xl text-center text-black font-bold">Welcome to the E-Bike Shop</h1>
```

Comment:
please adjust the font color and size here to be like in the design 

![Capture d’écran du 2023-11-03 11-42-50](https://github.com/khanh-devos/capstone-ebike-reservation-frontend/assets/59707859/ceca005f-ef1c-40ad-b36f-b2260a73ef0a)